contracts/Front/Roles.sol

```diff
@@ -73,6 +88,17 @@ contract Roles is Base {
         rolesEventEmitter.nodeOnboarded(msg.sender, msg.value, "", Consts.NodeType.APOLLO);
     }
 
+    function onboardAsApolloSafe(address nodeAddress) public payable {
+        require(canOnboard(nodeAddress, Consts.NodeType.APOLLO, msg.value));
+
+        nodeAddressesStore.addNode(msg.sender, nodeAddress);
+        apolloDepositStore.storeDeposit.value(msg.value)(msg.sender);
+        rolesStore.setRole(nodeAddress, Consts.NodeType.APOLLO);
+        validatorProxy.addValidator(nodeAddress, msg.value);
+
+        rolesEventEmitter.nodeOnboarded(msg.sender, msg.value, "", Consts.NodeType.APOLLO);
```

Comment:
I think first argument here should be nodeAddress. I haven't found  any calls to onboardAsApolloSafe so I dunno who is the sender.